### ISSUE TITLE
Question about LabelSmoothingLoss

### ISSUE THREAD

**Issue by AbnerCode** [THUcqb/KOBE#16]
Hi @qibinc ,

I am using your code. I found that you do not use the CrossEntropy loss, instead of LabelSmoothingLoss, so can you give some explanation, and your paper also not explain this thing.

Best wishes,

**Comment by qibinc** [comment]
Hi @AbnerCode ,

Thanks for pointing out this. Since first introduced by https://arxiv.org/pdf/1512.00567 in 2015, **label smoothing** has been commonly adopted to the training of many deep neural networks, such as GANs (https://arxiv.org/pdf/1606.03498.pdf). It is essentially cross entropy loss plus a small modification to encourage the model to be less confident (regularization). You can study the above two papers (search for label smoothing in file).

Hope this helps you.

Best,
Qibin
